tests/cpp/lp_namer/ProblemModifierTest.cpp

```diff
@@ -319,121 +355,159 @@ TEST_F(ProblemModifierTest, One_link_two_candidates_two_timestep) {
     const std::vector<ActiveLink>& links = linkBuilder.getLinks();
 
     auto problem_modifier = ProblemModifier();
-    math_problem = problem_modifier.changeProblem(std::move(math_problem), links, p_var_columns,p_cost_columns);
-
-    verify_columns_are(4);
-    verify_rows_are(5);
-    verify_row_is_of_type(0, 'L');
-    verify_row_is_of_type(1, 'G');
-    verify_row_is_of_type(2, 'L');
-    verify_row_is_of_type(3, 'G');
-    verify_row_is_of_type(4, 'L');
-
-    verify_elems_are(15);
-
-    verify_mat_val_is(0, 1);
-    verify_mat_val_is(1, -links.at(0).getCandidates().at(0).direct_profile(0));
-    verify_mat_val_is(2, -links.at(0).getCandidates().at(1).direct_profile(0));
-    verify_mat_val_is(3, 1);
-    verify_mat_val_is(4, links.at(0).getCandidates().at(0).indirect_profile(0));
-    verify_mat_val_is(5, links.at(0).getCandidates().at(1).indirect_profile(0));
-    verify_mat_val_is(6, 1);
-    verify_mat_val_is(7, -links.at(0).getCandidates().at(0).direct_profile(1));
-    verify_mat_val_is(8, -links.at(0).getCandidates().at(1).direct_profile(1));
-    verify_mat_val_is(9, 1);
-    verify_mat_val_is(10, links.at(0).getCandidates().at(0).indirect_profile(1));
-    verify_mat_val_is(11, links.at(0).getCandidates().at(1).indirect_profile(1));
-    verify_mat_val_is(12, 1);
-    verify_mat_val_is(13, -1);
-    verify_mat_val_is(14, -1);
-
-    verify_rhs_is(0, links.at(0)._already_installed_capacity * links.at(0).already_installed_direct_profile(0));
-    verify_rhs_is(1, -links.at(0)._already_installed_capacity* links.at(0).already_installed_indirect_profile(0));
-    verify_rhs_is(2, links.at(0)._already_installed_capacity * links.at(0).already_installed_direct_profile(1));
-    verify_rhs_is(3, -links.at(0)._already_installed_capacity* links.at(0).already_installed_indirect_profile(1));
-    verify_rhs_is(4, 0);
-
-    ASSERT_EQ(problem_modifier.get_candidate_col_id(cand1.name) , 2);
-    ASSERT_EQ(problem_modifier.get_candidate_col_id(cand2.name) , 3);
+    math_problem = problem_modifier.changeProblem(std::move(math_problem), links, p_var_columns,p_direct_cost_columns,p_indirect_cost_columns);
+    verify_columns_are(5);
+
+    const int P_LINK_id  = 0;
+    const int P_PLUS_id = 1;
+    const int P_MINUS_id = 2;
+    const int cand1_id= 3;
+    const int cand2_id= 4;
+
+    verify_column(P_LINK_id,P_LINK,'C',1,-1e20,1e20);
```

Comment:
we can "rename"
`0` into `ZERO` 
`1e20` into `PLUS_INF` 
`-1e20` into `MINUS_INF` 

the third argument, if I remember well corresponds to the  variable type
1=double (or continuous)
0=integer

we could name two constants:
`INTEGER_VAR`
`CONTINUOUS_VAR`
 Continuous or other, I dontremeber how the multisolver interface colls it

Reply:
third argument is objective. No new constant defined